app/models/miq_report/generator.rb

```diff
@@ -284,7 +295,11 @@ def _generate_table(options = {})
         :skip_counts      => true,
       )
 
-      rbac_opts[:extra_cols] = va_sql_cols unless va_sql_cols.nil? || va_sql_cols.empty?
+      if va_sql_cols.present? || custom_attributes_sql_cols.present?
```

Comment:
At a glance, this probably looks like redundant if checks, but basically we want to have `rbac_opts[:extra_cols` to be either not set, or a populated array.  Otherwise this fails in `ar_virtual.rb` with the over written `.select` with have code around.